beta/src/content/apis/react/useMemo.md

```diff
@@ -2,19 +2,342 @@
 title: useMemo
 ---
 
-<Wip>
-
-This section is incomplete, please see the old docs for [useMemo](https://reactjs.org/docs/hooks-reference.html#usememo).
-
-</Wip>
-
-
 <Intro>
 
+`useMemo` is a React Hook that returns a [memoized](https://en.wikipedia.org/wiki/Memoization) value.
+
 ```js
 const memoizedValue = useMemo(callback, [...dependencies])
 ```
 
 </Intro>
 
 <InlineToc />
+
+<Note>
+
+The difference between `useMemo` and [`useCallback`](apis/react/useCallback) is that `useMemo` returns a memoized value, and `useCallback` returns a memoized callback.
+
+</Note>
+
+- [Usage](#usage)
+  - [Skip expensive recalculations](#skip-expensive-recalculations)
+  - [Skip re-render](#skip-re-render)
+- [Reference](#reference)
+  - [`const memoizedValue = useMemo(callback, [...dependencies])`](#usememo)
+- [Troubleshooting](#troubleshooting)
+  - [Every time my component renders `useMemo` is triggered](#every-time-my-component-renders-useMemo-is-triggered)
+
+---
+
+## Usage {/*usage*/}
```

Comment:
Heads up: this is handled already with <InlineToc /> above now, so it's duplicated:

<img width="1188" alt="Screen Shot 2022-09-06 at 1 37 13 PM" src="https://user-images.githubusercontent.com/2440089/188702758-ef07a4b1-3dd2-4e1b-950c-e82737bd68d9.png">

Reply:
Thanks, will remove it.